atc/api/pipelineserver/reject_archived_handler_factory_test.go

```diff
@@ -0,0 +1,100 @@
+package pipelineserver_test
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/concourse/concourse/atc/api/pipelineserver"
+	"github.com/concourse/concourse/atc/db/dbfakes"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Rejected Archived Handler", func() {
+	var (
+		response *http.Response
+		server   *httptest.Server
+		delegate *delegateHandler
+
+		dbTeamFactory *dbfakes.FakeTeamFactory
+		fakeTeam      *dbfakes.FakeTeam
+		fakePipeline  *dbfakes.FakePipeline
+
+		handler http.Handler
+	)
+
+	BeforeEach(func() {
+		delegate = &delegateHandler{}
+
+		dbTeamFactory = new(dbfakes.FakeTeamFactory)
+		fakeTeam = new(dbfakes.FakeTeam)
+		fakePipeline = new(dbfakes.FakePipeline)
+
+		handlerFactory := pipelineserver.NewRejectArchivedHandlerFactory(dbTeamFactory)
+		handler = handlerFactory.RejectArchived(delegate.GetHandler(fakePipeline))
+	})
+
+	JustBeforeEach(func() {
+		server = httptest.NewServer(handler)
+
+		request, err := http.NewRequest("POST", server.URL+"?:team_name=some-team&:pipeline_name=some-pipeline", nil)
```

Comment:
ha, neat!

Reply:
Thought so too! I copied it from the scoped handler test in the same package 😅